### PR TITLE
Update to TileDB 2.13.0

### PR DESCRIPTION
**Pull request by ihnorton**

### CODE REVIEW

**Review by Shelnutt2 — APPROVED** [review]

**Review by leipzig — APPROVED** [review]

File: tiledb.rb [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-    sha256 "0e77650ab86cb65cd04f9f6fcb6d8bfc600fd59c03805c6b739a10e7b05e5821"
+    url "https://github.com/TileDB-Inc/TileDB/releases/download/2.13.0/tiledb-macos-x86_64-2.13.0-db00e70.tar.gz"
+    sha256 "d91cbe1e709d2b8e844dbcb51e29333573accf92b33a0a5a1ce1661c834ced44"
   end
 
   if OS.linux? && Hardware::CPU.intel?
-    url "https://github.com/TileDB-Inc/TileDB/releases/download/2.12.0/tiledb-linux-x86_64-2.12.0-ac8a0df.tar.gz"
-    sha256 "e27a51db6fa19b753cda3c9fca16950fa39126d48687c8f3df23091755df5ea0"
+    url "https://github.com/TileDB-Inc/TileDB/releases/download/2.13.0/tiledb-linux-x86_64-2.13.0-db00e70.tar.gz"
+    sha256 ""
```
Comment (shaunrd0) [inline tiledb.rb]: `324aecb43cbc16ae7bc0cded1b90e632c36d28b26e45f990922516e5af73428c` for Linux sha256?

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Official [Homebrew](https://brew.sh/) packages for TileDB.
 
 ## Install TileDB (from stable tap)
 
-The latest stable verison of [TileDB v2.12.0](https://github.com/TileDB-Inc/TileDB/releases/tag/2.12.0)
+The latest stable verison of [TileDB v2.12.0](https://github.com/TileDB-Inc/TileDB/releases/tag/2.13.0)
```
Comment (shaunrd0) [inline README.md]: Link still reads `TileDB v2.12.0`, but points to 2.13.0 release

**Review by shaunrd0 — APPROVED** [review]
Two small comments but otherwise :+1:

File: tiledb.rb (hunk 1 above)
Comment (ihnorton) [reply tiledb.rb]: Thanks

**Review by shaunrd0 — APPROVED** [review]

**Pull request merged**